packages/components/src/components/date-of-birth-picker/date-of-birth-picker.tsx

```diff
@@ -2,17 +2,17 @@ import React from 'react';
 import { toMoment } from '@deriv/shared';
 import DatePicker from '../date-picker';
 
-const DateOfBirthPicker = props => {
-    const [max_date] = React.useState(toMoment().subtract(18, 'years'));
-    const [min_date] = React.useState(toMoment().subtract(100, 'years'));
+const DateOfBirthPicker = (props: Omit<React.ComponentProps<typeof DatePicker>, 'display_format' | 'max_date' | 'min_date'>) => {
+    const max_date = toMoment().subtract(18, 'years');
+    const min_date = toMoment().subtract(100, 'years');
+
     return (
         <DatePicker
+            {...props}
             calendar_view='year'
             display_format='DD-MM-YYYY'
             max_date={max_date}
             min_date={min_date}
```

Comment:
```suggestion
            max_date={toMoment().subtract(18, 'years')}
            min_date={toMoment().subtract(100, 'years')}
```
nit-pick ) if we don't need a useState and nothing's breaking, then we can remove these 2 variables at line 6 & 7 because they are used just once here, so we can use the values directly 🙂